phpBB/develop/create_search_index.php

```diff
@@ -132,7 +132,7 @@
 
 $search->tidy();
 
-add_log('admin', 'LOG_SEARCH_INDEX_CREATED', $search_name);
+$phpbb_log->add('admin', $user->data['user_id'], $user->ip, 'LOG_SEARCH_INDEX_CREATED', time(), array($search_name));
```

Comment:
I didn't think about that. But I think `0` would be a better default value for `log_interface::add()`, as the documentation says it should be an int.